### PR TITLE
Fixes EZP-24739: ezFind IndexBoost not working in some locales

### PR DESCRIPTION
**Pull request by kuborgh-hgiesenow**
Link: https://jira.ez.no/browse/EZP-24739

### CODE REVIEW

**Comment by yannickroger** [comment]
I updated the title and added the link to the issue.

File: classes/ezsolrdoc.php [hunk 1]
```diff
@@ -118,7 +118,9 @@ function docToXML()
 
         if ($this->DocBoost !== null && is_numeric( $this->DocBoost ) )
         {
-            $this->DomRootElement->setAttribute( 'boost', $this->DocBoost );
+            // EZP-24739 - This will force the '.' as decimal separator and not depend on the locale
```
Comment (yannickroger) [inline classes/ezsolrdoc.php]: You don't need the issue number in the comment, we can find it thanks to git.

**Comment by yannickroger** [comment]
Looks good to me +1
You should update the commit message:  `Fix EZP-24739: ezFind IndexBoost not working in some locales` 

ping @paulborgermans

**Comment by paulborgermans** [comment]
+1 from me as well, thanks

**Comment by yannickroger** [comment]
@kuborgh-hgiesenow I can merge it as soon as you took care of the feedback. Feel free to ping me when it is done

**Comment by yannickroger** [comment]
I will take care of the feedback as it is now a support request :)

**Comment by yannickroger** [comment]
Thanks @kuborgh-hgiesenow

**Pull request closed without merging**